declarations.d.ts

```diff
@@ -0,0 +1,6 @@
+declare module "*.svg" {
```

Comment:
What was the reason behind including this file?

Reply:
Since react native does not support SVG's we are using a library which allows us to import SVG's like in react. This is needed for typescript to recognize the SVG's properly. As per the library documentation here https://github.com/kristerkari/react-native-svg-transformer#using-typescript